spec.html

```diff
@@ -46182,10 +46175,35 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
           1. Return _unfiltered_.
       </emu-alg>
       <p>The *"length"* property of this function is *2*<sub>𝔽</sub>.</p>
-      <emu-note>
-        <p>Valid JSON text is a subset of the ECMAScript |PrimaryExpression| syntax. Step <emu-xref href="#step-json-parse-validate"></emu-xref> verifies that _jsonString_ conforms to that subset, and step <emu-xref href="#step-json-parse-assert-type"></emu-xref> asserts that that parsing and evaluation returns a value of an appropriate type.</p>
-        <p>However, because <emu-xref href="#sec-runtime-semantics-propertydefinitionevaluation"></emu-xref> behaves differently during `JSON.parse`, the same source text can produce different results when evaluated as a |PrimaryExpression| rather than as JSON. Furthermore, the Early Error for duplicate *"__proto__"* properties in object literals, which likewise does not apply during `JSON.parse`, means that not all texts accepted by `JSON.parse` are valid as a |PrimaryExpression|, despite matching the grammar.</p>
-      </emu-note>
+
+      <emu-clause id="sec-ParseJSON" type="abstract operation">
+        <h1>
+          ParseJSON(
+            _text_: a String,
+          ): either a normal completion containing an ECMAScript language value or a throw completion
+        </h1>
+        <dl class="header">
+        </dl>
+        <emu-alg>
+          1. [id="step-json-parse-validate"] If StringToCodePoints(_text_) is not a valid JSON text as specified in ECMA-404, throw a *SyntaxError* exception.
```

Comment:
I had to reword this step otherwise ecmarkup was complaining that this AO can never return an abrupt completion.

Reply:
That's arguably a bug in ecmarkup, but I like this wording better anyway.